src/components/map/map-direction-tool.tsx

```diff
@@ -158,7 +158,7 @@ export class MapDirectionTool extends BaseComponent<IProps, IState> {
           <CurrentDirection>
             Direction =
             <Direction>{direction % 360}°</Direction>
-            from 0° (N)
+            from N
```

Comment:
Out of curiosity, is there room to spell out "North"?

Reply:
Probably.  "N" does tie back to what appears on the compass, but I'm willing to settle with what the story requested (at least until they tell us to change it!).